src/fiskaltrust.Middleware.Interface.Client.Soap/SoapPosFactory.cs

```diff
@@ -8,6 +8,16 @@ namespace fiskaltrust.Middleware.Interface.Client.Soap
     public static class SoapPosFactory
     {
         public static async Task<IPOS> CreatePosAsync(ClientOptions options)
+        {
+            var soapClientOptions = new SoapClientOptions
+            {
+                RetryPolicyOptions = options.RetryPolicyOptions,
+                Url = options.Url
```

Comment:
Should we specify the default value here

Reply:
I'd rather leave this in the class (as the default values for the properties), because we do it like that in other places as well